lib/js-file.js

```diff
@@ -588,6 +668,23 @@ JsFile.prototype = {
     },
 
     /**
+     * Executes callback for each token in token list.
+     *
+     * @param {Function} cb
+     * @private
+     */
+    _forEachToken: function(cb) {
```

Comment:
Nope, for iterating the list which changes during the iteration. I had problems with it. Because if you just use `forEach`, then `index` becomes invalid once you push items (like adding some whitespace before the token).